src/helpers/config.py

```diff
@@ -60,7 +60,7 @@ def check_db_connection(connection: Engine, db_type: str) -> Engine:
     """
     try:
         if connection:
-            with connection.connect() as conn:  # Use connection.connect() to get a Connection object
+            with connection.connect() as conn:
```

Comment:
The naming is a bit confusing to me. Why is the engine called `connection` and then a connestion is created and it is called `conn`?

Reply:
Fixed the naming. Its actually an engine object used to check if a connection is possible to test connection status. If its not, we just return a new engine else return the same engine. conn is the connection object to check our dummy execution.